.github/workflows/haskell.yml

```diff
@@ -129,6 +129,11 @@ jobs:
     - name: Build all
       run: cabal build all --enable-tests
 
+    # This step is temporary. We'll soon remove it and check instead
+    # that golden files have not been modified when creating them below.
+    - name: Test all
+      run: cabal test all --enable-tests --test-show-details=direct -j1
```

Comment:
I'm confused by the role this step has.  Can you elaborate what problem you are solving with this temporary workaround?

Reply:
There is more context [here on Slack](https://input-output-rnd.slack.com/archives/G011N23CEAE/p1713252179618169?thread_ts=1713211750.320289&cid=G011N23CEAE). Gist is that we were not checking that golden files are correct.